changelog/dmd.classinfo.dd

```diff
@@ -0,0 +1,10 @@
+Added .nameSig field to TypeInfo_Class in object.d
+
+This is a 16 byte md5 signature of the fully qualified name of the class.
+It is used to compare two classes for equality, rather than comparing the
+pointers with a fallback to doing a string compare on the name, which can
+be rather slow.
+
+The result is both druntime and phobos will need to be recompiled to be
```

Comment:
This is probably misleading if a user sees this in the changelog of a release

Reply:
Not sure how it's misleading. What do you suggest?

Reply:
I worry that someone may see this as a recommendation that they have to do something to druntime or similar, which won't be true because they're using said release.

So something more explicit about where any breakage would actually come about would better, that and that the size has changed.